modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -254,6 +254,12 @@ definition {
 		PortletEntry.save();
 	}
 
+	macro addObjectFolderViaAPI {
+		JSONObjectFolder.addFolder(
+			folderLabel = ${folderLabel},
+			folderName = ${folderName});
+	}
+
```

Comment:
I think we can just call the API macro directly instead of wrapping it in another macro.